packages/machine-extractor/package.json

```diff
@@ -12,7 +12,7 @@
     "@babel/parser": "^7.21.4",
     "@babel/traverse": "^7.21.4",
     "@babel/types": "^7.21.4",
-    "prettier": "^2.8.7",
+    "prettier": "^2.8.8",
```

Comment:
i believe that now machine-extractor removed its dependency on prettier so this could be removed altogether

Reply:
Good catch, removed